torchao/quantization/quant_api.py

```diff
@@ -599,7 +600,12 @@ def apply_int8_dynamic_activation_int4_weight_quant(
     if act_mapping_type == MappingType.ASYMMETRIC:
         input_quant_func = _int8_asymm_per_token_quant
     elif act_mapping_type == MappingType.SYMMETRIC:
-        input_quant_func = _int8_symm_per_token_quant
+        if isinstance(layout, MarlinQQQLayout):
+            input_quant_func = _int8_symm_per_token_quant
+        elif isinstance(layout, CutlassInt4PackedLayout):
```

Comment:
you can set zero_point_domain to be ZeroPointDomain.NONE(https://github.com/pytorch/ao/blob/f7f20e9e8ba28a4910efb62e7b18243143d8d855/torchao/quantization/quant_primitives.py#L78) so that zero_point is None